lib/fs/fs.ml

```diff
@@ -3,8 +3,12 @@ type file_contents = {
   offset : int;
 }
 
+type file_type =
+  | Text
+  | Binary
+
 type tree =
-  | File of string * file_contents lazy_t
```

Comment:
Haha, I probably introduced `lazy_t` when I wasn't that familiar with best practices.

Agree to use `Lazy.t` instead, especially if this is what the documentation suggests.